x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/endpoint_package_custom_extension/components/fleet_trusted_apps_card.tsx

```diff
@@ -63,8 +91,8 @@ export const FleetTrustedAppsCard = memo<PackageCustomExtensionComponentProps>((
             </h4>
           </EuiText>
         </EuiFlexItem>
-        <EuiFlexItem grow={false}>
-          <TrustedAppItemsSummary />
+        <EuiFlexItem style={{ alignItems: 'center', justifyContent: 'center' }}>
```

Comment:
I would strongly recommend to reaching into the style overrides as much as possible. This bites maintainers and developers in the long run almost always. I have seen a lot of bad things with it.

For example, this making an assumption that `EuiFlexItem`'s implementation will be a flex group or compatible and makes it tricker if we get regressions or issues if the implementation changes later.

I would instead try to use another EuiFlexGroup inner group here and style it using its regular attributes that Eui has given us through their docs:
https://elastic.github.io/eui/#/layout/flex


I would consider